django_project/ford3/views/campus_wizard.py

```diff
@@ -48,6 +48,8 @@ def get_context_data(self, form, **kwargs):
         ]
         context['campus'] = self.campus
         context['provider'] = self.provider
+        context['provider_logo'] = self.provider.provider_logo.url \
```

Comment:
can't you get the logo url from `self.provider`?

Reply:
`self.provider` is a model.
`provider_logo` is a field of the `provider` model.

Reply:
my question is: why do you need to pass `provider_logo` in the context? you're already passing the `provider`.
Can't you access the logo url in your template like this: 
```
provider.provider_logo.url
```

?

Reply:
the `provider_logo` exist only to serve `sidebar.html` page.
the page is outside the ford3, hence the `provider`  is unknown.

Reply:
what about the line above context['provider_logo']?

Reply:
the context of `provider` serves the ford3 pages.
in this case: `campus_form.html`.
every `provider_logo` in the codes serves only for sidebar.html as it doesn't recognize `provider`